### PR TITLE
2dPhysicsDemo: make it compile and run on Haiku

### PR DESCRIPTION
**Pull request by owenca**

### CODE REVIEW

**Comment by scottmc** [comment]
Maybe convert the readme.txt to a readme.md and add a screenshot of it in action?

**Comment by owenca** [comment]
Sure.

**Comment by owenca** [comment]
Done.

File: Source/Makefile [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+# The type of binary, must be one of:
+#	APP:	Application
+#	SHARED:	Shared library or add-on
+#	STATIC:	Static library archive
+#	DRIVER: Kernel driver
+TYPE = APP
+
+# 	If you plan to use localization, specify the application's MIME signature.
+APP_MIME_SIG =
```
Comment (humdingerb) [inline Source/Makefile]: Please always add the signature in Makefiles: application/x-vnd.2dPhysicsDemo

**Review by humdingerb — CHANGES_REQUESTED** [review]

**Comment by humdingerb** [comment]
Can you retake the screnshot to include the whole windo, please?

**Comment by owenca** [comment]
Yes. :)

File: Source/Makefile (hunk 1 above)
Comment (owenca) [reply Source/Makefile]: Oops! I forgot this time. :(

**Comment by owenca** [comment]
Fixed the Makefile. I will push the fixes to the GUI layout shortly after this PR is merged. Then I will upload a new screenshot with the window title.

**Review by humdingerb — APPROVED** [review]

**Comment by humdingerb** [comment]
Thanks!

**Pull request merged**